provisioning/plugins/corim-psa-decoder/decoder.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/veraison/veraison/provisioning/decoder"
+	plugin_common "github.com/veraison/veraison/provisioning/plugins/common"
+)
+
+const (
+	SupportedMediaType = "application/corim-unsigned+cbor; profile=http://arm.com/psa/iot/1"
+	PluginName         = "unsigned-corim (PSA_IOT profile)"
+)
+
+type Decoder struct{}
+
+func (o Decoder) Init(params decoder.Params) error {
+	return nil // no-op
+}
+
+func (o Decoder) Close() error {
+	return nil // no-op
+}
+
+func (o Decoder) GetName() string {
+	return PluginName
+}
+
+func (o Decoder) GetSupportedMediaTypes() []string {
```

Comment:
I am not sure I understand your proposal.  In particular, which "base decoder" are you referring to?

Reply:
Hi T, What I meant was that the supported GetSupportedMediaTypes () is more like a Veraison Feature, which Media Types it supports for decoding, so could rather than individual plugin implement it, could the provisioning/decoder/decoder_rpc.go Server method has the implementation of what Media Types Veraison supports ?

Reply:
I don't think this can be classified as a "Veraison feature" in the current architecture.

Under the current model the base feature provided by Veraison is the [provisioning REST API](https://github.com/veraison/veraison/tree/main/docs/api/endorsement-provisioning), which is in fact media-type agnostic.

Reply:
As far as I can see, supported media type is a combination of content type and profile ? and which profile, Veraison supports is it not based on Veraison Feature set?

Reply:
> [...] which profile Veraison supports is it not based on Veraison Feature set?

No, I don't think so.  We (Veraison) are agnostic as to the media type (+profile) a given deployment will be able to deal with.

Reply:
Its OK